Add .Net Core base parent image and rename base to parent in Node.js images

johnwatson484 opened the pull request:
https://eaflood.atlassian.net/browse/PSD-393

johnwatson484 committed:
Add .Net Core base parent image and rename base to parent in Node.js images.

https://eaflood.atlassian.net/browse/PSD-393

johnwatson484 committed:
Base renamed to Parent in Node.js

johnwatson484 committed:
.Net Core dockerfiles added

johnwatson484 committed:
Development images reference parent

djbingham reviewed (commented):
There are a few more changes of "base" -> "docker" required. Arguably, these belong in a separate PR but it might make sense to include them here as you've renamed the registry in the examples.

- Readme title: "FFC docker base" -> "FFC Docker parent"
- `ffc-node-parent/Dockerfile`: `LABEL` instruction uses the prefix `uk.gov.defra.ffc-node-base`
- `examples/Dockerfile.web` and `examples/Dockerfile.service` each have a comment before the final `CMD` line, which talks about "the parent base image" - this should be simply "the parent image"

johnwatson484 committed:
Additional base references removed

johnwatson484 commented:
> There are a few more changes of "base" -> "docker" required. Arguably, these belong in a separate PR but it might make sense to include them here as you've renamed the registry in the examples.
> 
> * Readme title: "FFC docker base" -> "FFC Docker parent"
> * `ffc-node-parent/Dockerfile`: `LABEL` instruction uses the prefix `uk.gov.defra.ffc-node-base`
> * `examples/Dockerfile.web` and `examples/Dockerfile.service` each have a comment before the final `CMD` line, which talks about "the parent base image" - this should be simply "the parent image"

Thanks, I've updated those instances too.  Makes sense to do them now as I renamed all other instances as part of this PR.  Just missed those ones.

djbingham reviewed (approved):
LGTM
